scipy/stats/_stats_py.py

```diff
@@ -6064,9 +6064,14 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
     or `b`, and the t-statistic is calculated. This process is performed
     repeatedly (`permutation` times), generating a distribution of the
     t-statistic under the null hypothesis, and the t-statistic of the observed
-    data is compared to this distribution to determine the p-value. When
-    ``permutations >= binom(n, k)``, an exact test is performed: the data are
-    partitioned between the groups in each distinct way exactly once.
+    data is compared to this distribution to determine the p-value.
+    Specifically, the p-value reported is the "achieved significance level"
+    (ASL) as defined in Chapter 15 of [3]_. Note that there are other ways of
+    estimating p-values using randomized permutation tests, for other options,
+    see the more general `permutation_test`.
```

Comment:
I thought I should add this note based on what I learned finishing up gh-13899 (see [here](https://github.com/scipy/scipy/pull/13899#issuecomment-986104041)). You can check the accuracy of this note against the code by comparing the Lines 6393-6399 against the note linked above and also the Efron's definition of the two-sided ASL: 
 
![image](https://user-images.githubusercontent.com/6570539/146094955-0e3079b0-5989-4c4c-ad0d-d9679f0b529c.png)

Reply:
For what it’s worth, in Efron’s more recent book, [Computer Age Statistical Inference](https://hastie.su.domains/CASI_files/PDF/casi.pdf), this test is defined with `>=`. 
![9FAC3ACE-2C5B-4513-9540-C2CC27D17BA2](https://user-images.githubusercontent.com/1953382/146112583-08226af0-004a-456b-a591-87f60e69a9d2.jpeg)

Reply:
Much better! I'll list that as the reference.

Reply:
FWIW in large samples the distinction with the equality will usually be irrelevant-assuming I'm understanding the context correctly. The difference would matter in small sample sizes and you might not be able to achieve the exact coverage probabilities without an additional layer of randomization. When you add the additional layer of randomization these are known as randomized hypothesis tests-you flip a coin w/probability heads =p where p is set to achieve the type I error probability you want or specified. I don't think it makes sense in this PR to include randomization though documenting the behavior should be sufficient.

Reply:
By *randomized* here, I'm referring to randomly resampled permutations as opposed to all possible permutations.